modules/specifying-role-arn-albo-sts.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/installing-albo-sts-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-role-arn-albo-sts_{context}"]
+= Specifying the role ARN for ALBO on an STS cluster
+
+The role ARN for the Operator needs to be passed to the Operator as an environment variable. You can use the dedicated input box in the OperatorHub web UI or specify it in the Subscription resource when installing the Operator by using the OpenShift CLI.
+
+.Procedure
+
+. Create the Operator subscription with the role ARN by running the following command:
```

Comment:
The command which creates just a subscription won't be of much help for the user because other OLM resources need to be created too (see [the github docs](https://github.com/openshift/aws-load-balancer-operator/blob/main/docs/install.md#operator-installation-on-sts-cluster)).
When I put Subscription YAML in the draft PR I thought of just giving the user an example to be inspired by rather than concrete instructions.

Reply:
Would adding the additional commands listed in those docs be appropriate here or do we need more explicit instructions?

Reply:
I'd rather give a YAML example of a `Subscription` object as an example. So the same code just without `cat <<EOF | oc apply -f -`, like this:
```yaml
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: aws-load-balancer-operator
  namespace: aws-load-balancer-operator
spec:
  channel: stable-v1
  name: aws-load-balancer-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
  config: 
    env:
    - name: ROLEARN
      value: <rolearn> 
```

Reply:
Changed, but now I'm wondering if this actually describes a procedure at all. This seems more like a reference than a set of steps to be followed.

Reply:
Ended up creating a procedure based on the commands listed here: https://github.com/openshift/aws-load-balancer-operator/blob/main/docs/install.md?plain=1#L20-L43